client/src/api/genomes.ts

```diff
@@ -0,0 +1,3 @@
+import { fetcher } from "@/api/schema";
+
+export const fetchGenomes = fetcher.path("/api/genomes").method("get").create();
```

Comment:
This file is not needed, we try to avoid using the "genomes" naming because it can be any reference database key nothing genome-specific (we need to keep the API endpoint with this name for backward compatibility).
So this was already available under `client/src/api/dbKeys.ts`